<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapMap</class>
 <widget class="QWidget" name="MapMap">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>456</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>692</width>
    <height>456</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>692</width>
    <height>456</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Палетка</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>456</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>692</width>
     <height>456</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>692</width>
     <height>456</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget
{
	border-image: url(:/Pyramid/fon_next_menu.png);
}</string>
   </property>
   <widget class="QPushButton" name="selectMap">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>70</y>
      <width>173</width>
      <height>152</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>173</width>
      <height>152</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>173</width>
      <height>152</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/vibor_k_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/vibor_k.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/vibor_k.png);;
     border-width: 3;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lasso">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>69</y>
      <width>173</width>
      <height>152</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>173</width>
      <height>152</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>173</width>
      <height>152</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/pribl_o_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/pribl_o.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/pribl_o.png);;
     border-width: 3;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="dayNight">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>173</width>
      <height>152</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>173</width>
      <height>152</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>173</width>
      <height>152</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/regim_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/regim.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/regim.png);;
     border-width: 3;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoPoint">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>260</y>
      <width>173</width>
      <height>152</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>173</width>
      <height>152</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>173</width>
      <height>152</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/pereiti_v_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/pereiti_v.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/pereiti_v.png);;
     border-width: 3;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="layers">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>260</y>
      <width>173</width>
      <height>152</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>173</width>
      <height>152</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>173</width>
      <height>152</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/sloi_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/sloi.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/sloi.png);;
     border-width: 3;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="statusVideo_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>7</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>22</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>карта</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="gridBut">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>173</width>
      <height>152</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>173</width>
      <height>152</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>173</width>
      <height>152</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/greed_on_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/greed_on.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/greed_on.png);
     border-width: 3;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/PyramidWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
